dashboard/getdata.py

```diff
@@ -169,6 +207,8 @@ def main():
         with contextlib.suppress(FileNotFoundError):
             os.remove("linux.csv")
```

Comment:
Since there are four files with the same remove now, how do you think about a for loop with a list for simplicity?